assets/js/extensions/shipping-methods/shared-components/sortable-table/index.tsx

```diff
@@ -249,7 +246,7 @@ export const SortableTable = ( {
 						items={ items }
 						strategy={ verticalListSortingStrategy }
 					>
-						{ data &&
+						{ data && data.length ? (
```

Comment:
data.length should be enough, no need to do data && data.length, also, make sure to cast data.length to bool